packages/rpc-core/src/rpc-methods/getTokenAccountsByDelegate.ts

```diff
@@ -0,0 +1,136 @@
+import { Base58EncodedAddress } from '@solana/addresses';
+
+import {
+    Base58EncodedBytes,
+    Base58EncodedDataResponse,
+    Base64EncodedDataResponse,
+    Base64EncodedZStdCompressedDataResponse,
+    Commitment,
+    DataSlice,
+    LamportsUnsafeBeyond2Pow53Minus1,
+    RpcResponse,
+    Slot,
+    TokenAccount,
+    U64UnsafeBeyond2Pow53Minus1,
+} from './common';
+
+type TokenAccountInfoBase = Readonly<{
+    /** indicates if the account contains a program (and is strictly read-only) */
+    executable: boolean;
+    /** number of lamports assigned to this account */
+    lamports: LamportsUnsafeBeyond2Pow53Minus1;
+    /** pubkey of the program this account has been assigned to */
+    owner: Base58EncodedAddress;
+    /** the epoch at which this account will next owe rent */
+    rentEpoch: U64UnsafeBeyond2Pow53Minus1;
+    /** the data size of the account */
+    space: U64UnsafeBeyond2Pow53Minus1;
+}>;
```

Comment:
I nudged on this before, but a lot of these methods are starting to redefine the same types over and over again, this one being [`GetAccountInfoApiResponseBase`](https://github.com/solana-labs/solana-web3.js/blob/master/packages/rpc-core/src/rpc-methods/getAccountInfo.ts#L16-L22) or more practically `AccountInfo`

Reply:
Yep agreed! I wanted to review `getTokenAccountsByOwner` before poking any further, but it looks like that one actually mirrors this basically exactly. I did put the new data structure here in `common`, and as part of that next PR I'll pull the common data structures there. I think whatever makes sense to extract from what will then be 5 methods counts as common now!

Reply:
Sounds like a good plan!